schema/test_queries.py

```diff
@@ -93,6 +93,14 @@ def rethink():
             container.stop()
             break
 
+def is_the_same_job_as(id_job, ref_job):
+    same = True
```

Comment:
careful defaulting to true

if ref_job is an empty iterable, this will incorrectly return true.